src/main/jhipster/src/main/webapp/i18n/en/germplasm-list.json

```diff
@@ -28,6 +28,52 @@
                     "attributes": "ATTRIBUTES"
                 }
             }
-        }
+      },
+      "actions": {
+          "import-list":"Import list"
+      },
+      "import": {
+          "header": "Import List",
+              "file": {
+                  "input.description": "Choose the file you would like to import in format: Excel",
+                  "validation": {
+                      "file.empty": "Wrong name for Observation sheet or the file has no data. Please verify and try again",
+                      "extensions": "Only the following file extensions are allowed: {{param}}",
+                      "header": "At least GID, GUID or DESCRIPTION column is required",
+                      "gid.format": "Some GID values are not a positive integer. Please correct them and try again",
+                      "unknown.column": "Some columns were not recognized in the system or are not relevant to the import list and were thus ignored ({{param}})"
+
+                  }
+            },
+          "multi": {
+              "matches": {
+                  "header": "Select match",
+                  "description": "No matches were found for the following records using <b>NAME: {{name}}</b> ({{matchNumber}} of {{unassignedCount}}). Please provide a mapping. Unmapped records will be omited."
```

Comment:
Is this right? Shouldn't it be "_Multiple matches were found.._?

Reply:
I changed it.